modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/type/EmailNotificationType.java

```diff
@@ -303,12 +303,79 @@ public void sendNotification(NotificationContext notificationContext)
 				).build(),
 				subject);
 
-			_sendEmail(
+			sendNotification(
```

Comment:
@pedro-oliveira446 could you please replace all occurences of _sendEmail method to sendNotification,

Thanks!